client/src/components/Footer.tsx

```diff
@@ -0,0 +1,17 @@
+import { Stack } from "react-bootstrap";
+
+function Footer() {
+  return (
+    <Stack
```

Comment:
Maybe use the html `footer` tag for the footer. It doesn't change it on the outside, but it's a good practice. You usually want to use semantic elements when you can.

Reply:
good catch